File: src/main/java/hu/frontrider/arcana/research/theory/TheoryRegistry.java
package hu.frontrider.arcana.research.theory;


import net.minecraft.item.ItemStack;
import thaumcraft.api.items.ItemsTC;

import java.util.ArrayList;

import static net.minecraft.init.Blocks.CACTUS;
import static net.minecraft.init.Blocks.PUMPKIN;
import static net.minecraft.init.Items.*;

public class TheoryRegistry {

    public static void init() {
        ArrayList<ItemStack> itemStacks = new ArrayList<>();

        itemStacks.add(new ItemStack(WHEAT_SEEDS));
        itemStacks.add(new ItemStack(POTATO));
        itemStacks.add(new ItemStack(POISONOUS_POTATO));
        itemStacks.add(new ItemStack(BEETROOT_SEEDS));
        itemStacks.add(new ItemStack(APPLE));
        itemStacks.add(new ItemStack(WHEAT));
        itemStacks.add(new ItemStack(MELON));
        itemStacks.add(new ItemStack(PUMPKIN));
        itemStacks.add(new ItemStack(MELON_SEEDS));
        itemStacks.add(new ItemStack(PUMPKIN_SEEDS));
        itemStacks.add(new ItemStack(BEETROOT));
        itemStacks.add(new ItemStack(CARROT));
        itemStacks.add(new ItemStack(CACTUS));
        CardGrow.options = itemStacks.toArray(new ItemStack[0]);

        itemStacks.clear();

        itemStacks.add(new ItemStack(BEEF));
        itemStacks.add(new ItemStack(CHICKEN));
        itemStacks.add(new ItemStack(MUTTON));
        itemStacks.add(new ItemStack(PORKCHOP));

        CardDissect.options = itemStacks.toArray(new ItemStack[0]);

        itemStacks.clear();
        itemStacks.add(new ItemStack(ItemsTC.brain));
        itemStacks.add(new ItemStack(ROTTEN_FLESH));
        itemStacks.add(new ItemStack(BONE));

        CardDissectDead.options = itemStacks.toArray(new ItemStack[0]);



    }
}

File: src/main/java/hu/frontrider/arcana/research/theory/TheoryRegistry.kt
package hu.frontrider.arcana.research.theory


import net.minecraft.init.Blocks.CACTUS
import net.minecraft.init.Blocks.PUMPKIN
import net.minecraft.init.Items.*
import net.minecraft.item.ItemStack
import thaumcraft.api.items.ItemsTC
import java.util.*

object TheoryRegistry {

    fun init() {
        val itemStacks = ArrayList<ItemStack>()

        itemStacks.add(ItemStack(WHEAT_SEEDS))
        itemStacks.add(ItemStack(POTATO))
        itemStacks.add(ItemStack(POISONOUS_POTATO))
        itemStacks.add(ItemStack(BEETROOT_SEEDS))
        itemStacks.add(ItemStack(APPLE))
        itemStacks.add(ItemStack(WHEAT))
        itemStacks.add(ItemStack(MELON))
        itemStacks.add(ItemStack(PUMPKIN))
        itemStacks.add(ItemStack(MELON_SEEDS))
        itemStacks.add(ItemStack(PUMPKIN_SEEDS))
        itemStacks.add(ItemStack(BEETROOT))
        itemStacks.add(ItemStack(CARROT))
        itemStacks.add(ItemStack(CACTUS))
        CardGrow.options = itemStacks.toTypedArray()

        itemStacks.clear()

        itemStacks.add(ItemStack(BEEF))
        itemStacks.add(ItemStack(CHICKEN))
        itemStacks.add(ItemStack(MUTTON))
        itemStacks.add(ItemStack(PORKCHOP))

        CardGrow.options = itemStacks.toTypedArray()

        itemStacks.clear()
        itemStacks.add(ItemStack(ItemsTC.brain))
        itemStacks.add(ItemStack(ROTTEN_FLESH))
        itemStacks.add(ItemStack(BONE))

        CardGrow.options = itemStacks.toTypedArray()

    }
}
